### ISSUE TITLE
spyder is not launching even after doing complete uninstall and reinstall of anaconda (python 3.9) 

### ISSUE THREAD

**Issue by sparshgarg23** [spyder-ide/spyder#18066]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

Spyder was working perfectly fine 1 hour ago,then all of a sudden it refused to launch,When I opened cmd and typed spyder I recieved pySide2 error as well as some pyQt binding.I didn't even do anything .I just closed spyder normally,and then tried reopening it.

### What steps reproduce the problem?
Platform Windows
1. From anaconda folder click spyder icon to launch spyder
2. Spyder refuses to launch
3. go to cmd type spyder 
4. Tried uninstalling and installing anaconda again

### What is the expected output? What do you see instead?
Spyder window should launch.But even after reinstalling anaconda latest ,and resetting spyder settings the window doesn't launch

```
```

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):
  File "C:\Users\spars\anaconda3\lib\site-packages\qtpy\__init__.py", line 209, in <module>
    from PySide import __version__ as PYSIDE_VERSION  # analysis:ignore
ModuleNotFoundError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\spars\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\spars\anaconda3\lib\site-packages\spyder\app\start.py", line 233, in main
    from spyder.app import mainwindow
  File "C:\Users\spars\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 42, in <module>
    requirements.check_qt()
  File "C:\Users\spars\anaconda3\lib\site-packages\spyder\requirements.py", line 44, in check_qt
    import qtpy
  File "C:\Users\spars\anaconda3\lib\site-packages\qtpy\__init__.py", line 215, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```

## Versions

* Spyder version: 5.1.5
* Python version: 3.9
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

### Dependencies

```
PASTE DEPENDENCIES HERE
spyder isn't opening so can't paste them
```

**Comment by ccordoba12** [comment]
Hey @sparshgarg23, thanks for reporting. This error usually appears because you installed `pyqt5` with `pip` at some point. 

Now you have to uninstall Anaconda and reinstall it following the instructions in [our video](https://www.youtube.com/watch?v=Ul79ihg41Rs) to be able to successfully run Spyder again.

Please try that and let us know if it works for you.

**Comment by sparshgarg23** [comment]
Carlos I have installed and uninstalled anaconda multiple times.
Did that for almost 5 times. 6th time also spyder didn't launch.So now i am using text editor.

Method to install and uinstall
windows ->program and feature uinstall anaconda3
once finished,install anaconda using binary

**Comment by sparshgarg23** [comment]
Step 2
![make_spyder_work_again_2](https://user-images.githubusercontent.com/34626942/171015743-0af48624-75b8-42cf-ba35-3fe48c36c3db.jpg)

**Comment by sparshgarg23** [comment]
![make_spyder_work_again_3jpg](https://user-images.githubusercontent.com/34626942/171016501-f4302542-10d2-42b4-a4b3-67313b5f29cb.jpg)

**Comment by ccordoba12** [comment]
As I told you above, please see our video to learn how to that properly. It contains additional instructions to do a successful reinstallation.

**Comment by sparshgarg23** [comment]
hmmm,quick question previously I had blender installed .Do you think that is somehow interfering with the installation?

**Comment by ccordoba12** [comment]
Yes, I do. I already told you above:

> This error usually appears because you installed pyqt5 with pip at some point.

**Comment by sparshgarg23** [comment]
I installed and uinstalled spyder as per the video link,but it's still not working.I will try to create virtual env ,and then get back to you

**Comment by sparshgarg23** [comment]
unfortunately creating virtual environment also didn't solve it
got an error 
```
(py39) C:\Users\spars>spyder
Traceback (most recent call last):
  File "C:\Users\spars\anaconda3\envs\py39\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\spars\anaconda3\envs\py39\lib\site-packages\spyder\app\start.py", line 233, in main
    from spyder.app import mainwindow
  File "C:\Users\spars\anaconda3\envs\py39\lib\site-packages\spyder\app\mainwindow.py", line 42, in <module>
    requirements.check_qt()
  File "C:\Users\spars\anaconda3\envs\py39\lib\site-packages\spyder\requirements.py", line 47, in check_qt
    if parse_version(actual_ver) < parse_version(required_ver):
  File "C:\Users\spars\anaconda3\envs\py39\lib\site-packages\pkg_resources\__init__.py", line 120, in parse_version
    return packaging.version.Version(v)
  File "C:\Users\spars\anaconda3\envs\py39\lib\site-packages\pkg_resources\_vendor\packaging\version.py", line 264, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object
```
```
Steps to reproduce error
1.conda create -n py39 python=3.9
2.conda activate py39
3.conda install -y spyder
4.Launch spyder using the icon added on start menu
5.in cmd type spyder?
```

**Comment by sparshgarg23** [comment]
it actually started working after i restarted my system not sure what the issue was here?
currently i am running spyder in a seperate environment.

**Comment by ccordoba12** [comment]
Great! Thanks for letting us know about it.

**Comment by brzezink** [comment]
@ccordoba12 
Thank you for your comment. It helped me to solve the problem quickly. However, please note that there are some typos. I believe it should be:
```
conda create -n spyder-env -c conda-forge spyder
conda activate spyder-env
spyder
```

**Comment by ccordoba12** [comment]
You're right, thanks. I updated my comment to fix my typo.

**Comment by Nancy-1458** [comment]
Hi Carlos, I have the same problem than brzezing but on Ubuntu 21.04. I had some difficulties with spyder so I uninstalled it and installed it again but it isn't launching now. Can you help me, please? Thanks

**Comment by ccordoba12** [comment]
Hey @Nancy-1458, a couple of comments for you:

* How did you install Spyder?
* Please open a system terminal, activate your environment (if necessary) and launch Spyder with the command `spyder`. The post here the possible error printed there.

**Comment by ntovarsolorzano** [comment]
Not sure if may help anyone but, in the video that @ccordoba12 shared, she says something very important. 
The issue is NOT with spyder but with Python itself that is glitched. 
In my case, Python3.10 was glitched, but, If i created any environment using Python3.11, or Python3.9 it would open normally. My mission was to find the directory of such faulty python3.10 installation and delete it. 

My solution was:
Browse to `%AppData%/Roaming/Python`
Delete the folder `python310`, then delete anaconda and re-install the latest version from their website. 
From this point on, when I created any new venv relying on python310 and install spyder, that first time it would download and install python3.10, then spyder would install over that python version, and then it would open normally. 

Note: In my case my `base` venv of anaconda was relying on Python3.10. Installing anaconda latest version moved it to Python3.11 and that solved my (base) venv issue of not being able to open spyder, but issue persisted if I created any other venv using Python3.10. 

Hope this helps. 
Blessings in Jesus name. 
Nelson.
